src/background.js

```diff
@@ -666,22 +666,24 @@ chrome.tabs.onReplaced.addListener(function(addedTabId, removedTabId){
 // Listening for Avira Autopilot remote control UI
 // The Scout browser needs a "emergency off" switch in case Privacy Badger breaks a page.
 // The Privacy Badger UI will removed from the URL bar into the menu to achieve a cleaner UI in the future.
-chrome.runtime.onMessageExternal.addListener(
```

Comment:
I think this is supported? https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/runtime/onMessageExternal